fe/fe-core/src/main/java/com/starrocks/rpc/BackendServiceProxy.java

```diff
@@ -67,9 +67,14 @@ private BackendServiceProxy() {
         if (Config.enable_brpc_share_channel) {
```

Comment:
What is the benefit of brpc_share_channel？

Reply:
If this is disable, BRPC will create a connection for each request, then there will be too many connection between FE and BE.